.gitignore

```diff
@@ -2,3 +2,6 @@
 *.egg-info
 **/__pycache__
 dist/*
+venv/*
+tests/data/pluto_disk_cgm_icm/*
+tests/data/pluto_orszag_tang/*
```

Comment:
these lines do not belong in `.gitignore`. If you mean to ignore these files locally, I suggest putting these lines in `.git/info/exclude` instead (this file is purely local and not meant to be committed)

Reply:
Okay. I'll do that. Do you wish to have all the files that I prepared for testing locally shared in a .zip file? I can readily do that and you can then decide on what to include and what to exclude out. I have around 9 different PLUTO runs with different number of dimensions and geometry combinations.

Reply:
that's great, feel free to share a link !

Reply:
[Here it is](https://indianinstituteofscience-my.sharepoint.com/:u:/g/personal/alankardutta_iisc_ac_in/EXg3BmC-SCpAi6ffgtn1zCwBUcg8g1icENRXEbOtGyD2DA?e=aM8uzi). All of these runs don't have `meta` files. But I can easily add these files to those that you think worth including in the repo.

Reply:
awesome, thank you. I'll give these a look tomorrow !